docs/cli.md

```diff
@@ -34,39 +34,51 @@ in which it is being run. Currently model analyzer supports 2 modes.
 ### Online Mode
 
 This is the default mode. When in this mode, Model Analyzer will operate to find
-the optimal model configuration for an online inference scenario. By default in 
-online mode, the best model configuration will be the one that maximizes 
-throughput. If a latency budget is specified to the [analyze subcommand](#subcommand-analyze) via 
-`--latency-budget`, then the best model configuration will be the one with the highest throughput in the given budget. 
+the optimal model configuration for an online inference scenario. By default in
+online mode, the best model configuration will be the one that maximizes
+throughput. If a latency budget is specified to the [profile subcommand](#subcommand-profile) via
+`--latency-budget`, then the best model configuration will be the one with the highest throughput in the given budget.
 
-In online mode the analyze and report subcommands will generate summaries specific to online inference. 
+In online mode the analyze and report subcommands will generate summaries specific to online inference.
```

Comment:
doesn't profile do this now as well?

Reply:
I switched this to say `"profile and report subcommands"`